### ISSUE TITLE
Asset versioning isn't working

### ISSUE THREAD

**Issue by aleserrags** [symfony/symfony#59263]
### Symfony version(s) affected

7.2.1

### Description

I'm following the course “Cosmic Coding with Symfony 7,” but asset versioning isn't working as shown in the course. I completed the course using version 7.2.1. There were no errors, but the file path remains normal, without a version. However, when I created a new project using version 7.1, it worked as expected.

### How to reproduce

use the {{ asset('images/starshop-logo.png') }} function in symfony 7.2.1

**Comment by xabbuh** [comment]
Can you create a small example application that allows to reproduce your issue?

**Comment by aleserrags** [comment]
I followed the step-by-step guide from the course "Cosmic Coding with Symfony 7", which uses Symfony 7.0.
But essentially, these were the steps:

1. symfony new my_project_directory --version="7.2.x-dev"

2. composer require twig

3. composer require symfony/asset-mapper

4. composer require symfony/asset

Use this function in the template (view):
{{ asset('images/starshop-logo.png') }}

output in 7.2.1 "/assets/images/starshop-logo.png"

output in 7.1 "/assets/images/starshop-logo-dd53028a1a21ed21ee9c09f812e7aa6d.png"

**Comment by nicolas-grekas** [comment]
And is the image displayed?
In dev mode, there is no need for versioning.
In prod mode, you need to run the asset-map:compile command to generate them.

**Comment by aleserrags** [comment]
> And is the image displayed? In dev mode, there is no need for versioning. In prod mode, you need to run the asset-map:compile command to generate them.

But in version 7.1, it works with versioning in dev mode.

**Comment by nicolas-grekas** [comment]
Why does it matter? If it works, then what's the issue?

**Comment by aleserrags** [comment]
> Why does it matter? If it works, then what's the issue?

The issue is related to version 7.2.1.

**Comment by nicolas-grekas** [comment]
I'm sorry but please try to answer my questions. I asked many yet they're still pending.

**Comment by aleserrags** [comment]
> I'm sorry but please try to answer my questions. I asked many yet they're still pending.

My issue is that asset versioning does not work with Symfony 7.2, but it works with version 7.1.

**Comment by nicolas-grekas** [comment]
Let me try again:

- are the images still displayed?
- in dev mode or in prod mode?
- why care if in dev mode and images are displayed?

**Comment by aleserrags** [comment]
**are the images still displayed?**
Yes

**in dev mode or in prod mode?**
dev mode

**why care if in dev mode and images are displayed?**
I'm just reporting this because I thought it might be an issue, as this wasn't the case in version 7.1
I'm learning Symfony.

**Comment by nicolas-grekas** [comment]
OK thanks. Then I think this is not an issue. If the images are displayed, all works: in dev mode, assets are served dynamically so that no versioning is needed.

**Comment by aleserrags** [comment]
Ok thanks!
